webcompat/static/css/src/variables.css

```diff
@@ -1,4 +1,6 @@
 :root {
+  --grid-edge-s: 9px;
```

Comment:
As we deal with compatibility issues, edge might be misleading? 😅
How about we change it to `outermargin` or something similar?